daru-io.gemspec

```diff
@@ -20,6 +27,37 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.post_install_message = <<-EOF
+
+*************************************************
+|        Thank you for installing daru-io!      |
+|                                               |
+|                      //                       |
+|                     //                        |
+|                    ||                         |
+|                    ||                         |
+|                    ||                         |
+|                oOOOOOOOOOo                    |
+|               ,         Oo                    |
+|              //|         |                    |
+|              \\\\|         |                    |
+|                | --- .-. |                    |
+|                |  |  | | |                    |
+|                | _|_ ._. |                    |
+|                `---------`                    |
+|                                               |
+|  Hope you love using daru-io! Also, consider  |
+|  using daru-view for various visualizations   |
+|  such as Tables and Charts plotting. Read     |
+|  the README for interesting use cases and     |
+|  examples.                                    |
+|                                               |
+|  Cheers!                                      |
+|  SciRuby Team                                 |
+*************************************************
+
+EOF
```

Comment:
Please-please-please don't do this!
As a commercial programmer, who does dozens of `bundle install` per day, I **TOTALLY HATE** this "nice stuff". It takes the screen place and instead of looking at list of some updated gems I should look at some irrelevant ASCII-art.
We'll remove post-install message from daru eventually.

This option (`post_install_message`) is for important stuff, like on ver. 2.0.1 it can say "If you are updating from 1.5, please do ...", not for fun!!!